components/tag/TagsInput.js

```diff
@@ -1,20 +1,32 @@
 import { useRef } from "react";
 import XMarkIcon from "@heroicons/react/20/solid/XMarkIcon";
 import Input from "../form/Input";
-
-export default function TagsInput({ tags, onTagAdd, onTagRemove }) {
+import Notification from "@components/Notification";
```

Comment:
How about using 'Alert' component here? Wouldn't it make more sense here since we are alerting user that tag limit has exceeded?

Reply:
Yes, I can use the alert component, but the problem is that when I use the alert component, it appears just above the input tag, not in the upper right corner like the Notification component. To achieve this, I have to make changes to the CSS of the alert component. I think that would not be a great idea to change the CSS or add some condition, as other components might be using the alert component.